### ISSUE TITLE
Compiling error when compiling RocksDB in the parent dir with GCC 

### ISSUE THREAD

**Issue by bjlemaire** [facebook/rocksdb#8386]
`make -j24` returns a compiling error when using GCC.
**WORKAROUND TO AVOID THIS ERROR : run `make -j24 USE_CLANG=1` to compile with Clang instead.**

### Expected behavior

Regular successful compiling message

### Actual behavior

```
error: inlining failed in call to always_inline ‘long long unsigned int _mm_crc32_u64(long long unsigned int, long long unsigned int)’: target specific option mismatch
 _mm_crc32_u64 (unsigned long long __C, unsigned long long __V)
 ^~~~~~~~~~~~~
util/crc32c.cc:360:21: note: called from here
   *l = _mm_crc32_u64(*l, LE_LOAD64(*p));
make: *** [Makefile:2276: util/crc32c.o] Error 1
```

### Steps to reproduce the behavior
Run "`make clean; make -j24`". Looks like this is a machine dependent error.

**Comment by mrambacher** [comment]
Can you give more details about what operating system and compiler version you are using?

**Comment by bjlemaire** [comment]
It looks like it was an issue related to my local setup (OS).

**Comment by pdillinger** [comment]
Fixed in #8389
